archive.sh

```diff
@@ -11,6 +11,24 @@ VERSION_RAW="$(sed -n 's|<version>\(.*\)</version>|\1|p' ./FinSearchUnified/plug
 VERSION="$(echo -e "${VERSION_RAW}" | tr -d '[:space:]')"
 echo "Version: ${VERSION}"
 
+echo $(git stash)
+
+git fetch --all --tags --prune
+
+TAG=$(git tag -l "v${VERSION}")
+
+# If tag is available we proceed with the checkout and zip commands
+# else exit with code 1
+if [[ -z "${TAG}" ]]
+then
+echo "[ERROR]: Tag v${TAG} not found"
+exit 1
+else
+echo "Checking out ${TAG} from master .."
+fi
+
+git checkout tags/${TAG} -b master
```

Comment:
* I'm not sure if this comment is correct. Wouldn't it create a new branch `master` having `tags/${TAG}` as base?
* Wouldn't `git checkout v${TAG}` do the right thing?